### DISCUSSION TITLE
Dark mode issue in production

### DISCUSSION THREAD

**Discussion by ahmed-meraneh** [filamentphp/filament#2461] [Help] [answered]
### Package

filament/filament

### Package Version

2.0

### Laravel Version

^9.0

### Livewire Version

^2.10

### PHP Version

^8.0

### Bug description

When there is a livewire request, the color of the card turn into gray

https://user-images.githubusercontent.com/94760825/168970852-d1d454f9-de2e-4f3f-a216-2ca4ce472c95.mov

**Comment by danharrin** [comment]
I cannot replicate this. I can explain what's happening though...

we register this Livewire persistent middleware in a service provider:

```php
Livewire::addPersistentMiddleware([
    DispatchServingFilamentEvent::class,
    MirrorConfigToSubpackages::class,
]);
```

and this ensures that dark mode is preserved across Livewire requests

so I'm tempted to say that this is a Livewire issue potentially...

**Comment by ahmed-meraneh** [comment]
It's really weird because locally everything works fine. Must be that or potentially a deployment issue from me

**Comment by danharrin** [comment, marked as answer]
As a quick fix, publish the config files for the form builder & table builder, and enable dark mode.

**Reply by ahmed-meraneh** [reply]
Yes that's work 

Thank you for your great job
